### PR TITLE
Clarifying private file version control in the intro

### LINKED ISSUE #1445: Private Files Doc Update
Re: [Private Files](https://pantheon.io/docs/articles/sites/private-files)
## Issue Description:

Reorganize doc to make it clear that there is a private directory under version control (directly beneath root of site's codebase) and a private directory not under version control (files/private symlinked to different locations for Drupal and WordPress)

## Suggested Resolution
- [x] Work with jessi to update the doc

### PR DESCRIPTION
**Pull request by JessiFischer**
Closes #1445

## Effect
PR includes the following changes:
-  Clarify uses for private dirs in codebase vs filesystem

## Remaining Work

@rachelwhitton please test
cc @nataliejeremy for copy edit
cc @ari-gold

### CODE REVIEW

File: source/docs/articles/sites/private-files.md [hunk 1]
```diff
@@ -6,10 +6,15 @@ category:
     - drupal
 keywords: drupal, wordpress, private files, files, private keys, private
 ---
-Pantheon provides two spaces for non-web-accessible data:   
+Pantheon provides two potential spaces for storing non-web accessible data. Deciding which one to use will depend on whether or not this data should be kept under version control.
+
+For non-versioned controlled data, such as private files containing passwords, store this in the private directory in the filesystem. The file system private directory location will be a bit different for Drupal and Wordpress sites.
+
 Drupal sites: `sites/default/files/private`
```
Comment (bmackinney) [inline source/docs/articles/sites/private-files.md]: these two lines (13&14) should be the same, except for the code-relative path.

File: source/docs/articles/sites/private-files.md [hunk 2]
```diff
@@ -6,10 +6,15 @@ category:
...
 ---
-Pantheon provides two spaces for non-web-accessible data:   
+Pantheon provides two potential spaces for storing non-web accessible data. Deciding which one to use will depend on whether or not this data should be kept under version control.
+
+For non-versioned controlled data, such as private files containing passwords, store this in the private directory in the filesystem. The file system private directory location will be a bit different for Drupal and Wordpress sites.
+
 Drupal sites: `sites/default/files/private`   
 WordPress sites: `~/files/private`=`wp-content/uploads/private`
 
+For version-controlled data, such as cloudhook scripts, you'll want to use the private directory located at the site root. In both Drupal and Wordpress, this lives in `code/private`
```
Comment (bmackinney) [inline source/docs/articles/sites/private-files.md]: `~/code/private/`

**Comment by rachelwhitton** [comment]
@JessiFischer @nataliejeremy I refactored this doc entirely, to align with recommendations made in the power users group and for better clarity.
cc @ari-gold

File: source/_docs/private-paths.md [hunk 3]
```diff
@@ -1,36 +1,52 @@
...
 categories: [developing]
 tags: [files]
 keywords: drupal, wordpress, private files, files, private keys, private
 ---
-Pantheon provides two spaces for non-web-accessible data:   
-Drupal sites: `sites/default/files/private`   
-WordPress sites: `~/files/private`=`wp-content/uploads/private`
+The Pantheon platform recognizes two distinct private directories which can be used for storing non-web accessible data.
+
+**Private Path for Valhalla (Filesystem)**
```
Comment (ari-gold) [inline source/_docs/private-paths.md]: Private Path for Files (not under version control)

File: source/_docs/private-paths.md [hunk 4]
```diff
@@ -1,36 +1,52 @@
...
+Drupal: `sites/default/files/private`   
+WordPress: `wp-content/uploads/private`
+
+**Private Path for Code**
```
Comment (ari-gold) [inline source/_docs/private-paths.md]: Private Path for Code (under version control)

File: source/_docs/private-paths.md [hunk 5]
```diff
@@ -1,36 +1,52 @@
...
+Drupal: `/private`   
+WordPress: `/private`
+
+Determining which path to use depends on whether or not the data should be tracked with Git as part of your site's codebase. For example, secret tokens or credentials for third party services should not be version controlled alongside your site's code.
```
Comment (ari-gold) [inline source/_docs/private-paths.md]: Can we move this paragraph up before the paths?

File: source/_docs/private-paths.md [hunk 6]
```diff
@@ -1,36 +1,52 @@
...
-Drupal sites: `sites/default/files/private`   
-WordPress sites: `~/files/private`=`wp-content/uploads/private`
+The Pantheon platform recognizes two distinct private directories which can be used for storing non-web accessible data.
+
+**Private Path for Valhalla (Filesystem)**
+Drupal: `sites/default/files/private`   
+WordPress: `wp-content/uploads/private`
+
+**Private Path for Code**
+Drupal: `/private`
```
Comment (ari-gold) [inline source/_docs/private-paths.md]: Can we combine Drupal and WordPress since it's the same path?

File: source/_docs/private-paths.md [hunk 7]
```diff
@@ -1,36 +1,52 @@
...
 <div class="alert alert-info" role="alert">
 <h4>Note</h4>
 If you have not already created these directories, you will need to do that first. Creating the folders can be done via SFTP or Git in Dev, and pushed to your Test and Live environments.</div>
 
-## Private Files and Uploads
-### Drupal
-
-These files will be web-accessible based on the access control rules that you set for your site and will use the following directory: `sites/default/files/private`
+## Private Path for Code
+Store data that should be version controlled, such as cloudhook scripts, within the `/private` directory at the root level of your site's codebase (same level as `index.php`). If you're connecting via SFTP, navigate into the `code` directory and upload files to `/private`. If you're connecting via Git, use the `/private` directory at the root level of your cloned repository. The private path for code is the same for both Drupal and WordPress sites.
```
Comment (ari-gold) [inline source/_docs/private-paths.md]: Cloudhook >> [Quicksilver](https://pantheon.io/docs/quicksilver/)

**Comment by rachelwhitton** [comment]
+1

**Pull request merged**